tests/test_vrf.py

```diff
@@ -842,8 +842,8 @@ def setup_bgp_with_loopback(self, duthost, ptfhost, cfg_facts):
         # deploy start script
         ptfhost.template(src="vrf/bgp_speaker/start.j2", dest="%s/%s" % (exabgp_dir, 'start.sh'), mode="u+rwx")
 
-        # kill exabgp if any
-        ptfhost.shell("pkill exabgp || true")
+        # Send SIGTERM to exabgp processes if any
+        ptfhost.shell("pkill -f exabgp || true")
```

Comment:
this will kill other exabgp session on the backend network, so it is not feasible.

Reply:
@lguohan: There is a backend network on the PTF host?

Reply:
yes

Reply:
Then I believe the proper fix here is for the VRF test to utilize the exabgp PyTest fixture for exabgp which is used by the PyTest version of the BGP speaker test.

Reply:
all exabgp should use the exabgp ansible module to manage. bgp_speaker should also be refactored.

Reply:
Is there any issue tracking this? I would like to make sure before I close this PR.

Reply:
I could not find an open issue, so I created one: https://github.com/Azure/sonic-mgmt/issues/1914